### DISCUSSION TITLE
find matched process in webui

### DISCUSSION THREAD

**Discussion by ChunelFeng** [google/perfetto#944] [Q&A] [answered]
hi, i use 

`SELECT * FROM slice where EXTRACT_ARG(arg_set_id, 'debug.msg_id') = 1732187030785000 order by ts` 

find instant_point, and then i want to filter by the process name by sql.

i cannot find the way here:  https://perfetto.dev/docs/analysis/sql-tables

need some help, thank you.

![Image](https://github.com/user-attachments/assets/99aa061d-7187-4150-a4dd-ba652b0f1484)

**Comment by LalitMaganti** [comment, marked as answer]
```
include perfetto module slices.with_context;


SELECT process_name FROM thread_slice where EXTRACT_ARG(arg_set_id, 'debug.msg_id') = 1732187030785000 order by ts
```

**Reply by LalitMaganti** [reply]
Filtering by process name is just filtering on the process_name column:
```
include perfetto module slices.with_context;


SELECT *
FROM thread_slice
where EXTRACT_ARG(arg_set_id, 'debug.msg_id') = 1732187030785000 and process_name = 'foooooo'
order by ts
```
